app/services/acuant/acuant_client.rb

```diff
@@ -50,5 +74,36 @@ def merge_facial_match_and_liveness_response(facial_match_response, liveness_res
         },
       )
     end
+
+    def check_results(post_response, instance_id)
+      if post_response.success?
+        fetch_doc_auth_results(instance_id)
+      else
+        failure(post_response.errors.first, post_response.to_h)
```

Comment:
Could we return `post_response` directly here?

Reply:
yup